src/backend/access/common/detoast.c

```diff
@@ -448,12 +448,7 @@ toast_fetch_datum(struct varlena *attr)
 	/* Must copy to access aligned fields */
 	VARATT_EXTERNAL_GET_POINTER(toast_pointer, attr);
 
-<<<<<<< HEAD
-	ressize = toast_pointer.va_extsize;
-	numchunks = ((ressize - 1) / actual_max_chunk_size) + 1;
```

Comment:
I think you've lost GPDB-specific changes like this made by the [commit](https://github.com/arenadata/gpdb/pull/1612/commits/fdb72798eb0488d18a80787b070c44c72e4d8efd#diff-d01da5e6bdde61f98612d82a2efa40edb6253969ec34e3424fb550b20883c918R444).